runelite-client/src/main/java/net/runelite/client/plugins/blastfurnace/BlastFurnacePlugin.java

```diff
@@ -50,12 +55,17 @@
 {
 	private static final int BAR_DISPENSER = NULL_9092;
 
+	private static final String FOREMAN_PERMISSION_PATTERN = "Okay, you can use the furnace for ten minutes. Remember, you only need half as much coal as with a regular furnace.";
```

Comment:
Since it's no longer a pattern, use a better var name.

```suggestion
	private static final String FOREMAN_PERMISSION_TEXT = "Okay, you can use the furnace for ten minutes. Remember, you only need half as much coal as with a regular furnace.";
```

Reply:
Resolved in 0fadcde